src/main/java/uk/co/bconline/ndelius/service/impl/UserEntryServiceImpl.java

```diff
@@ -267,6 +273,12 @@ public void save(String existingUsername, UserEntry user) {
 			val newDn = LdapNameBuilder.newInstance(getDn(newUsername)).build();
 			log.debug("Renaming LDAP entry from {} to {}", oldDn, newDn);
 			ldapTemplate.rename(oldDn, newDn);
+
+			// Send Domain event
+			HashMap<String, String> domainEventValues = new HashMap<>();
+			domainEventValues.put("fromUsername", existingUsername);
+			domainEventValues.put("toUsername", newUsername);
+			domainEventService.insertDomainEvent(HmppsDomainEventType.UMT_USERNAME_CHANGED, domainEventValues);
```

Comment:
Using `Map.of` here would mean you don't need to manually create and populate a HashMap.
```suggestion
			val additionalInformation = Map.of(
			    "fromUsername", existingUsername,
			    "toUsername", newUsername
			);
			domainEventService.insertDomainEvent(HmppsDomainEventType.UMT_USERNAME_CHANGED, additionalInformation);
```